projects/lwip/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Copyright 2023 J. Neuschäfer
```

Comment:
Need to give google the copyright. If you really don't want to do this, then put most of the code elsewhere and just have the code in this repo pull that code down and call it.

Reply:
Ok